pkg/serverconfig/config.go

```diff
@@ -253,6 +253,14 @@ func addCustomization(fs *flag.FlagSet, customization *Customization) {
 			klog.Fatalf("Could not marshal ConsoleConfig customization.developerCatalog.categories field: %v", err)
 		}
 	}
+
+	quickStarts, err := json.Marshal(customization.QuickStarts)
```

Comment:
Do we need to check if the quick starts stanza is nil?

Reply:
@spadgett updated the code to have a nil check